src/schema/core.yaml

```diff
@@ -1627,47 +1637,105 @@ slots:
 
   total_bases:
     range: integer
-    todos:
-      - this slot needs some basic textual annotations and constraints
+    minimum_value: 0
+    description: Total number of basepairs.
   members_id:
     range: string
     multivalued: true
-    todos:
-      - this slot needs some basic textual annotations and constraints
+    description: Names of the contigs that make up a metagenome-assembled genome.
+    close_mappings:
+      - GENEPIO:0100596
   bin_name:
     range: string
+    required: true
+    description: Name of the metagenome-assembled genome.
   number_of_contig:
+    description: Number of contigs
+    exact_mappings:
+      - GENEPIO:0000093
     range: integer
+    minimum_value: 0
   completeness:
+    description: Estimate of the completeness of the metagenome-assembled genome, estimated by a tool like CheckM.
     range: float
+    minimum_value: 0
   contamination:
+    description: Estimate of the completeness of the metagenome-assembled genome, estimated by a tool like CheckM.
     range: float
+    minimum_value: 0
   gene_count:
     range: integer
+    description: Number of genes.
+    minimum_value: 0
   bin_quality:
-    range: string
+    description: The quality of the metagenome-assembled genome based on MIMAG standards (https://doi.org/10.1038/nbt.3893).
+    range: BinQualityEnum
   num_16s:
+    description: Number of 16s sequences detected, a subunit of prokaryotic ribosomes.
     range: integer
+    minimum_value: 0
+    broad_mappings:
+      - NCIT:C105370
   num_5s:
+    description: Number of 5s seqeuences detected, a subunit of ribosomes.
     range: integer
+    minimum_value: 0
+    broad_mappings:
+      - OMIT:0013248
   num_23s:
+    description: Number of 23 seqeuences detected, a subunit of ribosomes.
+    minimum_value: 0
     range: integer
+    broad_mappings:
+      - OMIT:0013245
   num_t_rna:
+    description: Number of transfer transfer RNAs.
+    minimum_value: 0
     range: integer
+    broad_mappings:
+      - NCIT:C816
   gtdbtk_domain:
+    description: Taxonomic domain assigned by GTDB-Tk.
+    comments: https://doi.org/10.1093/bioinformatics/btz848
```

Comment:
`comments`  is `multivalued`, so it takes a list of values. `see_also` is a better choice for capturing a URL, and it is also `multivalued`. So for `gtdbtk_domain` I recommend

```yaml
gtdbtk_domain:
    description: Taxonomic domain assigned by GTDB-Tk.
    comments: https://doi.org/10.1093/bioinformatics/btz848
    range: string
    exact_mappings:
      - NCBITaxon:superkingdom
```